build.gradle

```diff
@@ -285,7 +285,8 @@ dependencies {
     // TEST Akka //
     testImplementation group: 'com.typesafe.akka', name: "akka-testkit_${scalaBinaryVersion}", version: akkaBinaryVersion
 
-//    scoverage "org.scoverage:scalac-scoverage-plugin_${scalaBinaryVersion}:1.4.8", "org.scoverage:scalac-scoverage-runtime_${scalaBinaryVersion}:1.4.8"
+    scoverage "org.scoverage:scalac-scoverage-plugin_2.12.12:1.4.8", "org.scoverage:scalac-scoverage-runtime_2.12:1.4.8"
```

Comment:
This should continue to use the `scalaBinaryVersion` so that we don't miss it in future upgrades

Reply:
we have two patterns here:
`scalac-scoverage-plugin_2.12.12` and `scalac-scoverage-runtime_2.12`
I can have two variables for that but there is no guarantee releases of component will keep same approach since currently is different

Reply:
@JustinPihony done

Reply:
That's fun..